Source/DafnyCore/Options/DafnyProject.cs

```diff
@@ -41,6 +44,16 @@ public class DafnyProject : IEquatable<DafnyProject> {
     col = 1
   };
 
+  [IgnoreDataMember]
+  public static IEnumerable<Option> targetOptions {
+    get {
+      var options = SinglePassCodeGenerator.Plugin
+        .GetCompilers(DafnyOptions.Default).Select(backend => new Option<string>(backend.TargetExtension));
```

Comment:
This is to fetch all the relevant options from the target backend for read/write from toml. Added a test-case using `module-name`.

Reply:
This is again a churn (or more like wasted effort). This was implemented because we concluded during the design that anything that goes in to CLI also needs to go into TOML. All this was added as REQUIRED feature. I'm happy to remove this, not so happy on the wasted effort.